tests/Unit/BoltFactoryTest.php

```diff
@@ -14,7 +14,7 @@
 namespace Laudis\Neo4j\Tests\Unit;
 
 use Bolt\connection\IConnection;
-use Bolt\enum\ServerState;
+use Bolt\protocol\ServerState;
```

Comment:
ServerState doesn't exist as class anymore. Protocol instance itself has property serverState.

Reply:
Hey. Looks like there was an older version (6.x) installed via `composer.lock`. Due to this, I updated code based on that older version. I'll will update to `7.0.1` and update these parts of the code. Thanks for the great catch :)